File: beecrowd/beecrowd_1257.c
#include <stdio.h>
#include <string.h>
int counter(char str[], int c)
{
    int s = 0;
    for (int i = 0; i < strlen(str); i++)
    {
        s += (str[i] - 65) + i + c;
    }
    return s;
}
int main()
{
    int t;
    scanf("%d", &t);
    while (t--)
    {
        int x;
        scanf("%d", &x);
        int line_counter = 0, sum = 0;
        while (x--)
        {
            char str[27];
            scanf("%s", &str);
            sum += counter(str, line_counter);
            line_counter++;
        }
        printf("%d\n", sum);
    }

    return 0;
}

File: beecrowd/beecrowd_1257.java
import java.util.Scanner;

public class beecrowd_1257 {
    public static void main(String[] args) {

        int t;
        Scanner sc = new Scanner(System.in);
        t = sc.nextInt();
        for (int i = 0; i < t; i++) {
            int x;
            x = sc.nextInt();
            int line_counter = 0, sum = 0;
            for (int j = 0; j < x; j++) {
                String str;
                str = sc.next();
                sum += counter(str, line_counter);
                line_counter++;
            }
            System.out.printf("%d\n", sum);

        }
    }

    public static int counter(String str, int c) {
        int s = 0;
        for (int i = 0; i < str.length(); i++) {
            s += (str.charAt(i) - 65) + i + c;
        }
        return s;
    }
}
